Python/ki/xKIChat.py

```diff
@@ -553,7 +553,7 @@ def AddChatLine(self, player, message, cFlags, forceKI=True):
                 mKIdialog.show()
         if player is not None:
             separator = "" if pretext.endswith(" ") else " "
-            chatHeaderFormatted = U"{}{}{}:".format(pretext, separator, unicode(player.getPlayerName()))
+            chatHeaderFormatted = U"{}{}{}:".format(pretext, separator, (player.getPlayerName(), "utf8"))
```

Comment:
You've changed the unicode object to a tuple, which will result in a nonsense chat header. If the conversion to a unicode object is not working, the `ptPlayer.getPlayerName` bindings should be fixed in the engine code.